Source/hydro/Make.package

```diff
@@ -18,12 +19,13 @@ endif
 
 ca_F90EXE_sources += advection_util_nd.F90
 
-ca_F90EXE_sources += ppm_nd.F90
+F90EXE_sources += ppm_nd.F90
```

Comment:
I don't think it matters, but was this intentional that you changed which make variable it gets added to?

Reply:
no, that was a mistake (and breaks megafortran).  I'll fix it.